App services cidr (#353)
couchbasecloud/terraform-provider-couchbase-capella: examples/app_services_cidr/README.md

# Capella App Services Allowed CIDR Example

> This example shows how to create and manage App Service Allowed CIDR blocks in Capella.

Reviewer: ```suggestion
This example shows how to create and manage allowed CIDR blocks for an App Service in Capella.
```

> This creates a new Allowed CIDR block in the selected Capella App Service and lists existing Allowed CIDR blocks in the App Service. It uses the App Service ID to create and list Allowed CIDR blocks.

Reviewer: ```suggestion
This creates a new allowed CIDR block in the selected Capella App Service and lists its currently allowed CIDR blocks. It uses the App Service ID to create and list allowed CIDR blocks.
```

data.capella_app_services_cidr.existing_allowedcidr: Reading...
capella_app_services_cidr.new_allowedcidr: Import prepared!
  Prepared capella_app_services_cidr for import
capella_app_services_cidr.new_allowedcidr: Refreshing state... [id=id=854cbdf0-8ae3-4a42-9227-59c52a5ab4f2,organization_id=ffffffff-aaaa-1414-eeee-000000000000,project_id=ffffffff-aaaa-1414-eeee-000000000000,cluster_id=ffffffff-aaaa-1414-eeee-000000000000,app_service_id=ffffffff-aaaa-1414-eeee-000000000000]
data.capella_app_services_cidr.existing_allowedcidr: Read complete after 1s

Import successful!

The resources that were imported are shown above. These resources are now in
your Terraform state and will henceforth be managed by Terraform.
```

In this section, we pass the IDs as a single comma-separated string.
The first ID in the string is the a CIDR block ID, which identifies the resource that we want to import.

> The second ID is the organization ID i.e. the ID of the organization to which the project belongs.

Reviewer: ```suggestion
The second ID is the organization ID, which identifies the organization with the project.
```

The third ID is the project ID i.e. the ID of the project to which the cluster belongs.

> The fourth ID is the cluster ID which identifies the cluster the App Service is associated with..

Reviewer: ```suggestion
The fourth ID is the cluster ID, which identifies the cluster associated with the App Service.
```

> The fifth ID is the App Service ID which identifies the App Service of the .

Reviewer: ```suggestion
The fifth ID is the App Service ID, which identifies the specific App Service we are working with.
```

```
$ terraform plan
╷
│ Warning: Provider development overrides are in effect
│ 
│ The following provider development overrides are set in the CLI configuration:
│  - hashicorp.com/couchbasecloud/capella in /Users/$USER/workspace/terraform-provider-capella
│ 
│ The behavior may therefore not match any released version of the provider and applying changes may cause the state to become incompatible with
│ published releases.
╵
data.capella_app_services_cidr.existing_allowedcidr: Reading...
data.capella_app_services_cidr.existing_allowedcidr: Read complete after 0s

Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # capella_app_services_cidr.new_allowedcidr will be created
  + resource "capella_app_services_cidr" "new_allowedcidr" {
      + audit           = {
          + created_at  = (known after apply)
          + created_by  = (known after apply)
          + modified_at = (known after apply)
          + modified_by = (known after apply)
          + version     = (known after apply)
        }
      + cidr            = "10.0.0.0/16"

>       + cluster_id      = "f499a9e6-e5a1-4f3e-95a7-941a41d046e6"

Reviewer 2: @mohammed-madi Remove all the original IDs and replace with the generic one- "ffffffff-aaaa-1414-eeee-000000000000", this is a public repo, we don't want to expose these IDs
